apps/storefront/src/pages/QuickOrder/index.main.test.tsx

```diff
@@ -3933,7 +3943,9 @@ describe('When backend validation feature flag is on', () => {
         expect(screen.getByText('ERROR-SKU-123')).toBeInTheDocument();
       });
 
-      const addToCartButton = screen.getByRole('button', { name: /Add 1 products to cart/i });
+      const addToCartButton = await screen.findByRole('button', {
```

Comment:
This selector was failing on one of the runs, replaced with `findByRole` to see if it is more stable.